bin/configs/cpp-boost-beast-client-petstore.yaml

```diff
@@ -0,0 +1,6 @@
+generatorName: cpp-boost-beast-client
+outputDir: samples/client/petstore/cpp-boost-beast/generated
```

Comment:
thanks for the PR

how did you test the output in this new folder?

I tried running `build-and-test.bash` but got errors.